libs/application/templates/health-insurance/src/forms/messages.ts

```diff
@@ -350,19 +369,19 @@ export const m = defineMessages({
   },
   successfulSubmissionMessage: {
     id: 'hi.application:successfulSubmission.message',
-    defaultMessage:
-      'Númer umsókninnar er **{applicationNumber}**. Tölvupóstur til staðfestingar hefur verið sendur. ',
+    defaultMessage: 'Númer umsókninnar er **{applicationNumber}**.',
     description:
       'Your application number is **{applicationNumber}**. A confirmation e-mail has also been sent. ',
```

Comment:
Not that it will affect anything right now but the description still has reference to that an e-mail will be sent